ui/basicTab/basicTab.scss

```diff
@@ -1,46 +1,52 @@
 /* --------- Advantages ------------ */
 
 /* This avoids spaces between the elements in the row */
-.advantage-row {
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-}
-
-.advantage-row>div {
-  display: inline-block;
-}
-
-.advantage-row.advantage-row> :first-child {
-  width: 120px;
-  padding-right: 4px;
-  text-align: right;
-}
-
-.advantages .count-placeholder {
-  height: 18px;
-  width: 40px;
-}
-
-.advantage-row>.CP {
-  display: inline-block;
-  text-align: center;
-  width: 26px;
-}
-
-.advantages .extraadvantagename {
-  width: 124px;
-  margin-top: -1px;
-}
-
-.advantages .extraadvantageCP.extraadvantageCP {
-  color: $cp-color;
-  width: 40px;
-  margin-top: -1px;
-}
-
-.advantages .extraadvantageCPdescription {
-  display: inline-block;
-  text-align: center;
-  width: 26px;
+.advantages {
+  .advantage-row {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+
+    >div {
+      display: inline-block;
+    }
+
+    > :first-child {
+      width: 120px;
+      padding-right: 4px;
+      text-align: right;
+    }
+
+    select.skillexpertise {
+      width: 40px;
```

Comment:
This style was hidden in `epicSkills.scss`, now here.
No styles change in this file, just a reorg.